pom.xml

```diff
@@ -178,6 +178,16 @@
           <artifactId>maven-surefire-plugin</artifactId>
           <version>2.22.1</version>
         </plugin>
+        <!-- Ensure that the Maven jar plugin runs before the Maven
+             shade plugin by listing the plugin higher within the file. -->
```

Comment:
I think this comment does not belong to here for two reasons: (1) Here we actually have `shade` before `jar` and (2) I think the order in the children's `<plugin>` section matters not here in `<pluginManagement>`.

Reply:
Makes sense and thanks! I have moved it appropriately